### PR TITLE
bump version to 0.6.0

### PR DESCRIPTION
**Pull request by ycxioooong**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -155,35 +90,33 @@ def gen_packages_items():
...
-            'build': parse_requirements('requirements/build.txt'),
-        },
-        zip_safe=False)
+setup(
+    name='mmcls',
+    version=get_version(),
+    description='OpenMMLab Image Classification Toolbox and Benchmark',
+    long_description=readme(),
+    author='OpenMMLab',
+    author_email='yangleidev@gmail.com',
```
Comment (hellock) [inline setup.py:99]: openmmlab@gmail.com

File: setup.py [hunk 2]
```diff
@@ -155,35 +90,33 @@ def gen_packages_items():
...
+    keywords='computer vision, image classification',
+    url='https://github.com/open-mmlab/mmclassification',
+    packages=find_packages(exclude=('configs', 'tools', 'demo')),
+    package_data={'mmcls.ops': ['*/*.so']},
```
Comment (hellock) [inline setup.py:103]: There is no `mmcls.ops`.

File: setup.py [hunk 3]
```diff
@@ -155,35 +90,33 @@ def gen_packages_items():
...
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'License :: OSI Approved :: Apache Software License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
```
Comment (hellock) [inline setup.py:111]: Add 3.8

File: setup.py [hunk 4]
```diff
@@ -155,35 +90,33 @@ def gen_packages_items():
...
+    ],
+    license='Apache License 2.0',
+    setup_requires=parse_requirements('requirements/build.txt'),
```
Comment (hellock) [inline setup.py:114]: Actually `build.txt` is unnecessary.

Comment (ycxioooong) [reply setup.py:114]: so ``setup_requires='['pytest-runner']``?

Comment (hellock) [reply setup.py:114]: We may just remove this line and build.txt.

File: setup.py [hunk 5]
```diff
@@ -155,35 +90,33 @@ def gen_packages_items():
...
+        'Programming Language :: Python :: 3.8',
+    ],
+    license='Apache License 2.0',
+    setup_requires=parse_requirements('requirements/build.txt'),
+    tests_require=parse_requirements('requirements/tests.txt'),
+    install_requires=parse_requirements('requirements/runtime.txt'),
+    extras_require={
+        'all': parse_requirements('requirements.txt'),
+        'tests': parse_requirements('requirements/tests.txt'),
+        'build': parse_requirements('requirements/build.txt'),
```
Comment (hellock) [inline setup.py:120]: Also remove this.

**Review by hellock — APPROVED** [review]

**Pull request merged**